sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -315,6 +316,8 @@ public int getCurrentVolume() {
   @Override
   public void setFullscreen(boolean isFullscreen) {
     _layout.setFullscreen(isFullscreen);
+    setChanged();
+    notifyObservers();
```

Comment:
I want to assume that we will eventually send multiple types of notifications eventually, and don't want to pigeonhole ourselves early